code/game/objects/items/weapons/storage/secure.dm

```diff
@@ -228,7 +228,7 @@
 //        Secure Safe
 // -----------------------------
 
-/obj/item/weapon/storage/secure/safe
+/obj/item/weapon/storage/secure/safe//Why the fuck isn't this a structure
```

Comment:
Because oldcoders were badcoders, but you can't change it in this PR. No meme comments